source/views/sis/components/recent-search/list.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import * as React from 'react'
+import {FlatList, StyleSheet, Text} from 'react-native'
+import {ListSeparator, ListRow} from '../../../components/list'
+import {NoticeView} from '../../../components/notice'
+import * as c from '../../../components/colors'
+
+type Props = {
+	onQueryPress: (query: string) => any,
+	queries: string[],
+}
+
+export class RecentSearchList extends React.PureComponent<Props> {
+	renderSeparator = () => <ListSeparator spacing={{left: 20}} />
+
+	renderItem = ({item}: {item: string}) => (
+		<ListRow
+			arrowPosition="none"
+			onPress={() => {
+				this.onPressRow(item)
+			}}
+		>
+			<Text style={styles.listItem}>{item}</Text>
+		</ListRow>
+	)
+
+	onPressRow = (query: string) => {
+		this.props.onQueryPress(query)
+	}
+
+	keyExtractor = (item: string) => item
+
+	render() {
+		const {queries} = this.props
+		return (
+			<FlatList
+				ItemSeparatorComponent={this.renderSeparator}
```

Comment:
Do we want these to be separated? Doesn't the App Store just use whitespace?

Reply:
![img_3ac303f527b1-1](https://user-images.githubusercontent.com/24640726/37379867-bb7a33f0-2703-11e8-9429-df18c0b51302.jpeg)

Reply:
then it's ok!

Reply:
sorry idk how to make that smaller

Reply:
It's fine

Reply:
Add a right: 20 to where you have left: 20 to add margin on the right too